lib/string_to_class.rb

```diff
@@ -0,0 +1,11 @@
+module StringToClass
+  ALLOWED_CLASSES = [Chapter, Heading, Commodity].freeze
```

Comment:
This is wayyyyy to defensive. We control the factory definition and if a person has access to commit changes to that we've got significant and various other problems

Reply:
that's because brakeman do not like it: see
https://stackoverflow.com/questions/23741259/brakeman-unsafe-reflection-method-constantize-called-with-model-attribute

anyway, 
I removed the module, and just using just a method now

The alternative would be to disable that check, but I guess it's better to stay safe and follow brakeman.

Reply:
I think this should move to the class_determinator